buildscripts/kokoro/windows.bat

```diff
@@ -15,9 +15,9 @@ mkdir grpc-java-helper
 @rem Install 7za
 @rem TODO(mattkwong): After Windows tests are no longer running on Jenkins, 7za
 @rem doesn't need to be installed and make_dependencies.bat can use unzip instead
-wget http://www.7-zip.org/a/7za920.zip
-unzip 7za920.zip
-mv 7za.exe grpc-java-helper
+@wget -q http://www.7-zip.org/a/7za920.zip >NUL
```

Comment:
Let's leave these echo'd. My main concern is loops and things that spew a lot. These are just a few lines. The `>NUL` for unzip is probably appropriate though.